test/soak_test/start-client.sh

```diff
@@ -0,0 +1 @@
+node map_soak_test.js $1 > client-out.log 2>client-err.log &
```

Comment:
Same as the member side script, we may need to differentiate client log names

Reply:
I've added pid to log file names. See 5da7e23ab5ea8e83ebf669c610b3b4b1b8f5f094